modules/administration/pages/maintenance-windows.adoc

```diff
@@ -0,0 +1,25 @@
+[[maintenance-windows]]
+= Maintenance Windows
+
+The maintenance windows feature in {productname} allows you to schedule actions to occur during your maintenance window period ahead of time, and to create repeating tasks.
+
+
+
+.Procedure: Creating a New Maintenance Calendar
+. Step the first
+. ...
+. Final step
+
+
+
+.Procedure: Creating a New Schedule
+. Step the first
+. ...
+. Final step
+
+
+
+.Procedure: Performing a Maintenance Update
```

Comment:
As a 3rd step, I'd say we should tell the user to assign the *Schedule* to a *System*(or *System*s). When the user does this, it means that they can't schedule the "restricted" actions anytime, but only inside of the maintenance windows defined by the *Schedule* (and *Calendar*).

As a 4th step, we could tell them to perform a package upgrade (any other "restricted" action would do, but I think a package upgrade is a good example) inside of maintenance window. The user just go to the UI for package upgrade (`System details -> Software -> Packages -> Upgrade`), selects the package(s) to upgrade and then selects the maintenance window from the UI.
(I'd also avoid the term *maintenance update*, since this is something that users have associated with the SUMA releases. The more precise caption would be "Performing a Package Upgrade inside a Maintenance Window).